tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -47,12 +54,7 @@ def test_fields(self):
         self.assertEqual(str(uut), "qwertzuiop")
         self.assertEqual(uut.marker, ("##", "#", "#"))
         self.assertEqual(uut.range, None)
-
-    def test_not_implemented(self):
-        not_implemented = DocumentationComment(
-            "some docs", "nolang", "doxygen", None, None, None)
-        with self.assertRaises(NotImplementedError):
-            not_implemented.parse()
+        self.assertEqual(uut.metadata, python_doxygen_metadata)
```

Comment:
this is to make sure that the `.metadata` is working properly. All of the other properties have been done the same too.